examples/LLM_Workflows/NER_Example/lancedb_module.py

```diff
@@ -0,0 +1,104 @@
+from typing import Union
+
+import lancedb
+import numpy as np
+import pyarrow as pa
+from datasets import Dataset
+from datasets.formatting.formatting import LazyBatch
+from sentence_transformers import SentenceTransformer
+
+
+def db_client() -> lancedb.DBConnection:
+    """the lancedb client"""
+    return lancedb.connect("./.lancedb")
+
+
+def _write_to_lancedb(
+    data: Union[list[dict], pa.Table], db: lancedb.DBConnection, table_name: str
+) -> int:
+    """Helper function to write to lancedb.
+
+    This can handle the case the table exists or it doesn't.
+    """
+    try:
```

Comment:
Is this the best way to do this with lance? No "does table exist"?

Reply:
yep.